src/controllers/AuthLocalController.js

```diff
@@ -0,0 +1,52 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import status from '../config/status';
+import { User } from '../queries';
+import * as helper from '../helpers';
+
+dotenv.config();
+
+/**
+ * [Users description]
+ */
+export default class AuthLocalController {
+  /**
+   * @param  {object} req
+   * @param  {object} res
+   * @return {object} return an object containing the confirmation message
+   */
+  static async sendEmail(req, res) {
+    const { email } = req.body;
+    const result = await User.findOne({ email }); // check if the email exist
+    if (Object.keys(result).length <= 0) {
+      return res.status(404).send({
+        message: 'email not found..'
+      });
+    }
+
+    await helper.sendMail(email); // send mail
```

Comment:
@luc-tuyishime this should be wrapped by an error handler.

Reply:
we have an async handler middleware that catches errors.

Reply:
how has it been connected to this `sendMail` function?

Reply:
I call the middleware in the routes.
the path to the route: routes/auth/local.js

Reply:
👌